guides/doc-Managing_Hosts/topics/proc_registering-a-host-to-project-using-the-global-registration-template.adoc

```diff
@@ -56,7 +56,7 @@ Optional: If the *Registration* feature is not enabled on your {SmartProxy}, ent
 
 . From the *Operating System* list, select the operating system of hosts that you want to register.
 
-. Optional: If you want to register hosts through {SmartProxy}, from the *{SmartProxy}* list, select the {SmartProxy} to register hosts to.
+. From the *{SmartProxy}* list, select the {SmartProxy} to register hosts through {SmartProxy}.
```

Comment:
Copying my comment from #386:
```suggestion
. From the *{SmartProxy}* list, select the {SmartProxy} to register hosts through. You must select the internal {SmartProxy} if you do not want to use an external {SmartProxy}.
```
How about making it explicit like that?

Reply:
Fine with me.

Reply:
You can either click **Commit suggestion**, or manually apply it and commit if you like. Then I will merge.